hvplot/ui.py

```diff
@@ -339,8 +360,7 @@ class hvPlotExplorer(Viewer):
 
     labels = param.ClassSelector(class_=Labels)
 
-    # Hide the geo tab until it's better supported
-    # geo = param.ClassSelector(class_=Geo)
+    geo = param.ClassSelector(class_=Geo)
```

Comment:
I noticed a problem with geo and dataframe last week https://github.com/holoviz/hvplot/pull/810#issuecomment-1717392222. 

Have you looked at how this affects DataFrame explorer?

Reply:
I can reproduce using:
```python
import pandas as pd
import hvplot.pandas
import cartopy.crs as ccrs
import holoviews as hv

hv.extension("bokeh")

points = ccrs.Mercator().transform_point(-88, 40, ccrs.PlateCarree())
df = pd.DataFrame({"x": [-9796115.18980811], "y": [4838471.398061159]})

kwargs = {
    "geo": True,
    "features": {"coastline": "110m"},
}
df.hvplot.points("x", "y", **kwargs)
```

Reply:
I think it was because projection was not set.

https://github.com/holoviz/hvplot/assets/15331990/0b703e26-249c-4d94-8733-539ed284759f